packages/store/src/secure-store/keytar-store.ts

```diff
@@ -58,7 +58,7 @@ export class KeytarStore implements SecureStore {
                         result[String(entry.account)] = JSON.parse(entry.password);
```

Comment:
I debugged this code an checking that `entry. account` is defined should be sufficient. 
I think this needs more investigation